services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -46,6 +46,10 @@ import { GenericErrorPage } from '../Errors/GenericErrorPage'
 import styles from './SubmissionSummary.module.scss'
```

Comment:
This new column only supposed to appear on the main submission summary right? Or is supposed to appear on past revision summaries as well?  If so, that is separate file right now `SubmissionRevisionSummary`